py/redrock/fitz.py

```diff
@@ -252,6 +259,10 @@ def fitz(zchi2, redshifts, target, template, nminima=3, archetype=None, use_gpu=
                                                  solve_matrices_algorithm=template.solve_matrices_algorithm,
                                                  use_gpu=use_gpu)
 
+        #- Penalize chi2 for negative [OII] flux; ad-hoc
+        if hasattr(template, 'OIItemplate'):
+            zzchi2 += calc_negOII_penalty(template.OIItemplate, zzcoeff)
+
```

Comment:
I don't think this is right. `OIItemplate` is defined above as a numpy array but not as an attribute of the `template` object. Or am I missing something?

Reply:
Please ignore this comment; I saw after I wrote this that `OIItemplate` was being set in `redrock.templates` (and that the code above was vestigial and not needed).

Reply:
Thanks for catching that and removing it..  I had meant to remove that and forgot.  Moving it into the Templates class was so that there would be a single place defining the wavelength range and the concept of the OIItemplate.